tensorflow_serving/g3doc/setup.md

```diff
@@ -72,7 +72,12 @@ the `tensorflow-serving-api` PIP package using:
 pip install tensorflow-serving-api
 ```
 
-Note: The TensorFlow Serving Python API only supports Python 2.7. It does not support Python 3.
+Note: The TensorFlow Serving Python API
+[is only published for Python 2](https://pypi.python.org/pypi/tensorflow-serving-api/1.0.0),
```

Comment:
Let's link to the package, not a specific version. Just https://pypi.python.org/pypi/tensorflow-serving-api